time-tracker/src/components/second-counter.jsx

```diff
@@ -0,0 +1,18 @@
+import { useEffect } from "react";
+import { useDispatch } from "react-redux";
+import { runOncePerSecond } from "../redux/reducers.jsx";
+
+export default function SecondCounter() {
```

Comment:
Please avoid adding components whose sole purpose is adding logic without rendering anything